roles/pulp_webserver/tasks/firewalld.yml

```diff
@@ -20,15 +20,15 @@
         enabled: true
 
     - name: Accept HTTP connections on port {{ pulp_webserver_http_port }}
-      firewalld:
+      ansible.posix.firewalld:
         port: '{{ pulp_webserver_http_port }}/tcp'
         permanent: true
         immediate: true
         state: enabled
         zone: '{{ pulp_firewalld_zone | default(omit) }}'
 
     - name: Accept HTTPS connections on port {{ pulp_webserver_https_port }}
-      firewalld:
+      ansible.posix.firewalld:
```

Comment:
Try out this syntax:
```
+++ b/roles/pulp_webserver/meta/main.yml
@@ -23,3 +23,5 @@ galaxy_info:
     - pulpcore
 dependencies:
   - pulp_common
+collections:
+  - ansible.posix
```

It may cause the collection to be installed as a dependency.

I am almost certain it will eliminate the need to specify the FQCN.

Reply:
it did work! 🎉